VSTS/Tasks/SonarQubeScannerMsBuildBegin/SonarQubeHelper.ps1

```diff
@@ -0,0 +1,334 @@
+# When passing arguments to a process, the quotes need to be doubled and   
```

Comment:
There are also tests for the 2 tasks and for the common file in https://github.com/Microsoft/vsts-tasks/tree/master/Tests/L0